microsoft-edge/devtools-guide-chromium/console/utilities.md

```diff
@@ -63,904 +64,791 @@ These commands only work by entering them directly into the DevTools **Console**
-
----
 
 <!-- ====================================================================== -->
 ## Recently selected element or JavaScript object
 <!-- planned new heading to troubleshoot: -->
 <!-- ## $0 - $4 -->
 
 <!-- summary to bubble up: -->
-Returns a recently selected element or JavaScript object.
+`$0`, `$1`, `$2`, `$3`, and `$4` return recently selected elements or JavaScript objects.
 
 <!-- add'l info -->
-`$0` returns the most recently selected element or JavaScript object, `$1` returns the second most recently selected one, and so on.  The `$0`, `$1`, `$2`, `$3`, and `$4` commands work as a historical reference to the last five DOM elements that were inspected within the **Elements** tool, or the last five JavaScript heap objects that were selected in the **Memory** tool.
+`$0` returns the most recently selected element or JavaScript object, `$1` returns the second most recently selected one, and so on.  The `$0`, `$1`, `$2`, `$3`, and `$4` variables work as a historical reference to the last five DOM elements that were inspected within the **Elements** tool, or the last five JavaScript heap objects that were selected in the **Memory** tool.
 
-### Syntax
+#### Syntax
 
 ```javascript
 $0
 ```
 
-### Example
+#### Example
 
-In the following figure, an `img` element is selected in the **Elements** tool.  In the **Console** drawer, `$0` has been evaluated and displays the same element:
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
 
-![The $0 command](./utilities-images/console-image-highlighted-$0.png)
+1. In the **Elements** tool, select an element, for example an `<input>` element.
 
-The following image shows a different element selected in the same webpage.  The `$0` now refers to the newly selected element, while `$1` returns the previously selected element:
+1. Open the **Console** tool in the **Quick View** panel, at the bottom of DevTools.
 
-![The $1 command](./utilities-images/console-image-highlighted-$1.png)
+1. Type `$0` in the **Console** tool, and then press **Enter**.
 
-<br/><br/>
+   The `$0` variable returns the selected `<input>` element:
+
+   ![The result of evaluating the $0 command in the Console](./utilities-images/image-highlighted-$0.png)
+
+1. Select a different element in the **Elements** tool, for example a `<div>` element.
+
+1. In the **Console** tool, type `$0`, and then press **Enter**.
+
+   `$0` refers to the newly selected element, which is the `<div>` element.
+
+1. Type `$1`, and then press **Enter**.
+
+   `$1` refers to the previously selected element, which is the `<input>` element:
+
+   ![The result of evaluating the $0 and the $1 commands](./utilities-images/image-highlighted-$1.png)
 
----
 
 <!-- ====================================================================== -->
 ## Query selector
 <!-- planned new heading to troubleshoot: -->
 <!-- ## $(selector) -->
```

Comment:
fyi @captainbrosset - how to put special punctuation in subheadings and then link to the special subheading:

background re: the hidden comment - after entering the hidden comments, realized that you never have to guess or manually construct the # internal destination string; you can just retrieve it empirically from the rendered webpage.  So, a couple approaches to putting punctuation commands in subheadings and linking to them eg from ToC at top of page: 

* Put punctuation commands in a ## or #### subheading, eg `## Query selector - $(selector)`.  Push commit.  After doc build, view rendered page.  Extract the URL for the subheading.

* In VS Code, with Learn Auth Pack extensions-set installed, to create an internal link, type (# and then select the desired special-punctuation subheading.

Reply:
I applied this change, in a commit, to show syntax in headings